File: somatorio.java
class Main {
    public static void main(String[] args) {
      int[] conjuntoNumerico = {1, 2, 7, 4, 5};
          int resultado = somatorio(conjuntoNumerico);
  
          System.out.println("O somatório é: " + resultado);
    }
  
        public static int somatorio(int[] numeros) {
          int soma = 0;
          for (int i = 0; i < numeros.length; i++) {
              soma += numeros[i];
          }
          return soma;
      }
  }

File: somatorio.js
function somatorio(numeros) {
    let soma = 0;
    for (let i = 0; i < numeros.length; i++) {
        soma += numeros[i];
    }
    return soma;
}

// Exemplo de uso
let conjuntoNumerico = [1, 2, 7, 4, 5];
let resultado = somatorio(conjuntoNumerico);

console.log("O somatório é:", resultado);